labs/lab-4-jquery/script.js

```diff
@@ -0,0 +1,62 @@
+var party;
+var width;
+// var table;
+
+$(document).ready(function() { // not able to get this to quite work with jQuery
+   $(".form").hide();
+   var resturant = new FloorPlan(3, 3);
+   FloorPlan.render();
+   Table.render();
+});
+
+class Party {
+  constructor(partyName, partySize, partyContact) {
+    this.partyName = partyName;
+    this.partySize = partySize;
+    this.partyContact = partyContact;
+  }
+}
+
+class Table {
+  constructor(tableNum, status, party, available) {
+    this.tableNum = tableNum;
+    this.status = status;
+    this.party = null;
+    this.available = true;
```

Comment:
You've got both `this.status` AND `this.available`. Probably only need one or t'other?